components/Item/index.js

```diff
@@ -0,0 +1,50 @@
+import React, {useState} from 'react'
+import { View, Text, Button, TextInput } from 'react-native'
+import './index.styl'
+import { observer } from 'startupjs'
+
+export default observer(function Item(props) {
+
+  const [edit, setEdit] = useState(false)
+  const [input, setInput] = useState(props.input)
+
+  const Edit = () => {
+    setEdit(edit => !edit)
+  }
+
+    return(
+        <View styleName='container'>
+
+            <View styleName='greenButton'>
+                <Button
+                    title='C'
+                    color='green'
+                    onPress={props.completed}
+                />
+            </View>
+
+            <View>
+              {edit ?
+                <TextInput
+                  value={input}
+                  onSubmitEditing={() => {
+                    props.edit(props.item.id, input)
+                    Edit()
+                  }}
+                  onChangeText={input => setInput(input)}
+                /> : <Text styleName={props.state ? 'completed' : 'display'}>
+                  {props.item} [date:{props.date}]
```

Comment:
@TimmyByDeOverly item? bad name for a field that contains a title of a todo) or is it a whole object (todo)?